src/languages.h

```diff
@@ -9,89 +9,89 @@
 #include "userConfig.h"
 
 #if (LANGUAGE == 0) // DE
-#if (DISPLAYTEMPLATE <= 4)
-    static const char *langstring_set_temp = "Soll:  ";
-    static const char *langstring_current_temp = "Ist:   ";
-    static const char *langstring_brew = "Brew: ";
-    static const char *langstring_uptime = "Uptime:  ";
-#endif
-#if (DISPLAYTEMPLATE >= 20)  //vertical templates
-    static const char *langstring_set_temp_rot_ur =      "S: ";
-    static const char *langstring_current_temp_rot_ur =  "I: ";
-    static const char *langstring_brew_rot_ur =          "B: ";
-#endif
-
-static const char *langstring_offlinemode = "Offline";
-#if TOF == 1
-static const char *langstring_waterempty = "Wasser leer";
-#endif
-
-static const char *langstring_wifirecon = "Wifi reconnect:";
-static const char *langstring_connectwifi1 = "1: Verbinde WLAN:";
-static const char *langstring_nowifi[] = {"Kein ", "WLAN"};
-
-static const char *langstring_error_tsensor[] = {"Fehler, Temp: ", "Temp.-Sensor ueberpruefen!"};
+    #if (DISPLAYTEMPLATE <= 4)
```

Comment:
funny, it put the intendation fot the #if behainde an #if, but can´t set it right for the content :D

Reply:
Yes, because the `#if` are prepocessor macros and have nothing to do with the actual code. I find the general use of preprocessor stuff for `if(){}` problematic, but we can easily change that - and I understand you are working to get rid of them anyway...